### PR TITLE
9.0 add account invoice company search and group by

### PR DESCRIPTION
**Pull request by susport**
This module extends the functionality of account to search
and group by to company.

### CODE REVIEW

**Comment by jjscarafia** [comment]
:+1:  very usefull, filtering and grouping by company on invoices should be native

File: account_invoice_company_search/README.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Account Invioce Company Search
```
Comment (feketemihai) [inline account_invoice_company_search/README.rst:6]: Invoice...not Invioce

File: account_invoice_company_search/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Account Invioce Company Search",
```
Comment (feketemihai) [inline account_invoice_company_search/__openerp__.py:22]: The same, Invoice instead on Invioce

File: account_invoice_company_search/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    'website': 'https://odoo-community.org/',
+    'license': 'AGPL-3',
+    "category": "Accounting",
+    'depends': [
+        'account'
+        ],
+    'data': [
+        'views/invoice_view.xml'
+        ],
+    'demo': [],
```
Comment (feketemihai) [inline account_invoice_company_search/__openerp__.py:34]: remove empty tags

File: account_invoice_company_search/README.rst [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+============
+
+To install this module, you need to:
+
+  Just install the module
+
+Configuration
+=============
+
+To configure this module, you need to:
```
Comment (feketemihai) [inline account_invoice_company_search/README.rst:22]: You can remove this, just leave No Configuration Needed..

File: account_invoice_company_search/README.rst [hunk 5]
```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Account Invioce Company Search
+==============================
+
+This module adds possibility of filtering and grouping by company on invoices.
+The user has to be Multi company
```
Comment (feketemihai) [inline account_invoice_company_search/README.rst:10]: The user needs to be in Multi Company group

**Review by feketemihai — CHANGES_REQUESTED** [review]

**Pull request closed without merging**